src/main/java/assets/css/AcesUp.css

```diff
@@ -15,142 +15,4 @@ font-weight: bold;
 .cardLocation{
 width: 100px;
 background-color:#F0F0F0;
-}
```

Comment:
This messed up the whole CSS. 
Please update the version first before pulling it to the repository